docs/installation/configuration-files.rst

```diff
@@ -24,3 +24,19 @@ SECRET_KEY
    chars = 'abcdefghijklmnopqrstuvwxyz0123456789!@#$%^&*(-_=+)'
    print(''.join(random.choice(chars) for i in range(50)))
 
+logging
+    By default Pulp logs at an INFO level to syslog. Pulp also comes with a `console` handler.
+    Additional handlers can be defined and used like so:
+
+.. code-block:: yaml
+   :linenos:
+
+   logging:
```

Comment:
What about moving this to the server.yaml file? We have all of the defaults there as commented items, so there probably should be a 'handlers' section to the commented things. This example of how to configure I think would be more useful in the commented server.yaml file so they can see the example right when they go to configure it.